msgcntr/messageforums-app/src/webapp/jsp/discussionForum/area/dfTemplateOrganize.jsp

```diff
@@ -126,10 +126,12 @@
       <h:form id="revise">
 		
 			<!--jsp/discussionForum/area/dfTemplateOrganize.jsp-->
-        <sakai:tool_bar_message value="#{msgs.cdfm_default_template_organize}" />
```

Comment:
Is this sakai:tool_bar_message used anywhere else now that you've removed it from these pages? If not, should we get rid of it too?